src/components/main-form.js

```diff
@@ -9,7 +9,7 @@ import numeral from 'numeral';
 import styled, { css } from "react-emotion";
 import { Input } from 'reactstrap';
 import { FV, PMT } from 'formulajs/lib/financial';
-import { getAnnualResultsByOption } from './getAnnualResultsByOption'
+import { getMonthlyResultsByOption } from './getMonthlyResultsByOption'
```

Comment:
I'm not used to seeing files named as functions, is that a thing?  If so we should probably move it out of the `components` folder and put it in a `utils` folder or something along those lines. I would even name the file something like `mortgage-utils.js` and add in more useful functions in there but don't mind if you prefer to name it like the actual function as long as we keep it consistent

Reply:
Yeah I think the file name is ok for now, and if want to put more functions in the same file, then we can rename in the future. I'll move it out tho